### PR TITLE
Feat: replace icons with Material Icons

### LINKED ISSUE #145: [Bug]: Theme icon button looks like settings cog in darkmode.
### Issue description

When in lightmode, the theme icon is a moon that looks consistent with other websites. When clicked and we are now in dark mode it transitions to a sun (which could represent lightmode), but unfortunately it looks like a settings cog.

### Expected behaviour

Theme icon is consistent with other websites, and doesn't look like a settings cog in any states.

### Actual behaviour

Theme icon in darkmode looks like settings cog.

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

1. switch to darkmode

### Errors

_No response_

### Duplicate declaration

- [X] I have searched the issues tracker for this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by evavirseda**
# Description of change

Replace existing icons with Material Icons.
This PR aims to close #145 .

### CODE REVIEW

File: client/src/app/components/Modal.tsx [hunk 1]
```diff
@@ -35,10 +33,13 @@ class Modal extends Component<ModalProps, ModalState> {
...
                 >
                     {this.props.icon === "dots" && (
-                        <DotsIcon />
-                    )}
+                        <span className="material-icons">
+                            more_horiz
+                        </span>)}
                     {this.props.icon === "info" && (
-                        <InfoIcon />
+                        <span className="material-icons">
```
Comment (begonaalvarezd) [inline client/src/app/components/Modal.tsx:40]: There is a vertical misalignment here 🙏🏼 You can see an example on the identity page
image

File: client/src/app/components/Header.tsx [hunk 2]
```diff
@@ -99,7 +94,9 @@ class Header extends Component<HeaderProps, HeaderState> {
                                 >
                                     <div className="label">Utilities</div>
                                     <div className="icon">
-                                        <DropdownIcon />
+                                        <span className="material-icons">
```
Comment (begonaalvarezd) [inline client/src/app/components/Header.tsx:97]: The line height of the navbar elements (explorer, visualizer, utilities) now also create a bit of visual misalignment, see the utilities chevron
image

File: client/src/app/components/Modal.tsx (hunk 1 above)
Comment (evavirseda) [reply client/src/app/components/Modal.tsx:40]: Screenshot from 2022-03-01 12-48-44

Resolved

File: client/src/app/components/Header.tsx (hunk 2 above)
Comment (evavirseda) [reply client/src/app/components/Header.tsx:97]: Screenshot from 2022-03-01 12-43-47

Resolved

**Review by begonaalvarezd — COMMENTED** [review]
@evavirseda what about the footer icons? Can we replace those as well? wdyt?

**Comment by evavirseda** [comment]
@begonaalvarezd Social icons like these are not part of Material Icons

File: client/public/index.html [hunk 3]
```diff
@@ -1,32 +1,35 @@
...
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,600;1,400;1,600&display=swap" rel="stylesheet">
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-  </body>
-</html>
+
+<head>
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/public/index.html]: Resolved

File: client/public/index.html [hunk 4]
```diff
@@ -1,32 +1,35 @@
...
+  <link href="https://webassets.iota.org/fonts/css/inter.css" rel="stylesheet" type="text/css" media="all">
+  <link href="https://webassets.iota.org/fonts/css/metropolis.css" rel="stylesheet" type="text/css" media="all">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,600;1,400;1,600&display=swap"
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/public/index.html]: Resolved

**Pull request merged**